### ISSUE TITLE
QA policy is confusing about what job and performer is it is related to

### ISSUE THREAD

**Issue by rgevaerd** [zerocracy/zerocracy.github.io#42]
Examples of confusions: 
1) Reading [§42 “QA Rules.”](http://www.zerocracy.com/policy.html#42) and [§31 “QA Bonus.”](http://www.zerocracy.com/policy.html#31) I thought all listed were applied to PR and Ticket as a whole. For example if "The code reviewer didn’t find any issues in the source code changes of a pull request and just accepted it;" as per §42 then both DEV and REV would receive no payment as per §31.
2) You can see in comments at [datum#294](https://github.com/zerocracy/datum/issues/294), StuporHero also got confusions due to current form these policies are written.

Suggestion to make the QA policy clearer: split these policy items separating by PR QA and ticket QA. Examples of changes using this mindset:
1) the rule "The code reviewer didn’t find any issues in the source code changes of a pull request and just accepted it;" would be in the PR QA rules but not in ticket QA rules. 
2) Bonus for DEV would be explained in ticket QA bonus, while bonus for REV will be explained in PR QA bonus.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@rgevaerd[/z](https://www.0crat.com/u/rgevaerd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@rgevaerd job performer is a user who is performing a job:
when job is an issue, then performer is a developer who is fixing this bug/issue
when job is a pull-request, then performer is a code-reviewer who is finding mistakes in the code.

**Comment by rgevaerd** [comment]
@g4s8 yes, I know that now, because I asked on telegram group and they explained to me. But I believe that the way it is currently described in the [website](http://www.zerocracy.com/policy.html#42) will cause many new people that are starting to learn about zerocracy to understand it wrong, as already happened. So I opened this issue for it to be fixed by making such information clearer.
